Library/OcBootManagementLib/OcBootManagementLib.c

```diff
@@ -544,27 +551,39 @@ OcLaunchAppleBootPicker (
   } else if (FirmwareUI->Revision != APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL_REVISION) {
     DEBUG ((
       DEBUG_INFO,
-      "OC: Launch Apple picker incompatible FirmwareUI protocol revision %u != %u\n",
+      "OC: Unlock Apple picker incompatible FirmwareUI protocol revision %u != %u\n",
       FirmwareUI->Revision,
       APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL_REVISION
       ));
     Status = EFI_UNSUPPORTED;
   }
 
   if (!EFI_ERROR (Status)) {
+    UIVars = &BASE_CR (FirmwareUI, APPLE_FIRMWARE_UI_LAYOUT, UIProtocol)->UIVars;
```

Comment:
Does it work with 32-bit efi? In terms of at least not breaking.

Reply:
I think this was also related to the packed error - so resolving